CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ Deprecated Features
 
 New Features
 ------------
+- Use circular weight function in HSM adaptive moments code. (#917)
```

Comment:
I recommend making clear this is optional - something like "Add option to use circular..."